cpp/include/cudf/stream_compaction.hpp

```diff
@@ -252,6 +253,34 @@ std::unique_ptr<table> unique(
   null_equality nulls_equal           = null_equality::EQUAL,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief Create a new table without duplicate rows.
+ *
+ * Given an `input` table_view, each row is copied to the output table if the corresponding
+ * row of `keys` columns is distinct (no other equivalent row exists in the table).
+ *
+ * The order of elements in the input table is not reserved when copied to the output.
+ *
+ * Performance hints:
+ * - Always use `cudf::unique` instead of `cudf::distinct` if the input is pre-sorted.
```

Comment:
"Always" is a strong statement. I have trouble believing it is true.

Reply:
To clarify, this is from my previous stream compaction refactor work.